pkg/apis/cluster/v1alpha1/label.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Label structure for instancepool node labels
+type Label struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
How come TypeMeta and ObjectMeta? Do labels get named/labels/annotations/kinds etc? (same applies for Taints)

Reply:
I'll confess that this is just coming from [Volume](https://github.com/jetstack/tarmak/blob/master/pkg/apis/cluster/v1alpha1/volume.go#L30) and probably doesn't need to be here :flushed:

Reply:
Fixed in 1fab9f97